453. Minimum Moves to Equal Array Elements.py

```diff
@@ -0,0 +1,16 @@
+class Solution(object):
+    def minMoves(self, nums):
+        """
+        :type nums: List[int]
+        :rtype: int
+        """
+        small = nums[0]
+        output = 0
+        
+        for num in nums:
+            small = min(small, num)
+            
+        for num in nums:
```

Comment:
you can change this to something like:
`return sum([num - small for num in nums])`.